api/types/swarm/service.go

```diff
@@ -34,9 +34,10 @@ type ServiceSpec struct {
 	UpdateConfig   *UpdateConfig `json:",omitempty"`
 	RollbackConfig *UpdateConfig `json:",omitempty"`
 
-	// Networks field in ServiceSpec is deprecated. The
-	// same field in TaskSpec should be used instead.
-	// This field will be removed in a future release.
+	// Networks specifies which networks the service should attach to.
+	//
+	// Deprecated: This field is deprecated since v1.44 and will be removed in a future release. The Networks field in
+	// TaskSpec should be used instead.
```

Comment:
FWIW; we won't be able to remove this from our code until we stop supporting API v1.43, but we can stop supporting it on newer API versions.


I guess for swarm services, there may be some additional challenges as the spec is stored in the cluster, or is that not an issue for this part of the code? 🤔 

/cc @dperny